pennylane/pulse/parametrized_hamiltonian_pytree.py

```diff
@@ -53,12 +53,12 @@ def from_hamiltonian(H: ParametrizedHamiltonian, *, dense: bool = False, wire_or
         make_array = jnp.array if dense else sparse.BCSR.fromdense
 
         if len(H.ops_fixed) > 0:
-            mat_fixed = make_array(qml.matrix(H.H_fixed(), wire_order=wire_order))
+            mat_fixed = make_array(matrix(H.H_fixed(), wire_order=wire_order))
         else:
             mat_fixed = None
 
         mats_parametrized = tuple(
-            make_array(qml.matrix(op, wire_order=wire_order)) for op in H.ops_parametrized
+            make_array(matrix(op, wire_order=wire_order)) for op in H.ops_parametrized
```

Comment:
I wonder if we could actually just use `H.H_fixed().matrix()` instead, as it would have less baggage than `qml.matrix`.